lib/irb/pager.rb

```diff
@@ -1,5 +1,7 @@
 # frozen_string_literal: true
 
+require 'reline'
```

Comment:
Q: do you think pager should be part of Reline?
If we position Reline as Ruby's default terminal application utility library, it kinda makes sense for it to have a pager class too 🤔

Reply:
I don't think the basic pager should be in Reline, because it's mainly about command process names(less, more) and options(-R, -X).
The added preview part is IRB specific that we can't assume pretty_print not printing a log.
If we can assume no log while pritty-printing, there is an easy way: just stream the output to pager io.

Making a pure-ruby pager library (key handling, rendering, scrolling, etc): I think it is very interesting.